### DISCUSSION TITLE
http requests from my site

### DISCUSSION THREAD

**Discussion by sgrossiP** [blakeblackshear/frigate#10092] [General]
hello, i am trying to consume from my web endpoints from the http section that worked before i upgrade from 0.12 to 0.13 but im getting 502 or cors deppending depending on which 
can i do something on this or i need to get back to 0.12?

**Comment by NickM-27** [comment]
examples and more info is needed, there is no reason the http api would not work in 0.13

**Comment by sgrossiP** [comment]
im not sure why but i keep getting this from my web running on localhost or production (on frigate webs page on :5000 works fine)
:4200/#/canchaId/8/dia/27/hora/17/mes/2/establecimiento/Reconquista/token/sportadmin/eid/2:1 Access to XMLHttpRequest at 'http://localhost:5000/vod/bahiados/start/1709076650/end/1709076680/index.m3u8' from origin 'http://localhost:4200' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource

the weird thing is that other endpoints works in 0.13 and vod_ts works too in 0.12 from my web
i know i cant modify http.py but do you have a clue? im kinda lost

ty

**Comment by sgrossiP** [comment]
Nick, nevermind, i saw in the pull requests history that the nginx.conf file has been changed and the headers has been removed, this was in 0.12
 aio threads;
            vod hls;

            secure_token $args;
            secure_token_types application/vnd.apple.mpegurl;

            add_header Access-Control-Allow-Headers '*';
            add_header Access-Control-Expose-Headers 'Server,range,Content-Length,Content-Range';
            add_header Access-Control-Allow-Methods 'GET, HEAD, OPTIONS';
            add_header Access-Control-Allow-Origin '*';
            add_header Cache-Control "no-store";
            expires off;
i think i need to create a proxy that can handle that so i dont need to modify frigate core 
by the way, im working on a comercial application that is using frigate, please let me know  if i need to talk with you about this
thanks for reading

**Reply by NickM-27** [reply]
I am not the author, questions about commercial use should go to @blakeblackshear but the license is visible on the main repo page

**Reply by blakeblackshear** [reply]
It's MIT licensed, so there is no restriction on commercial use.
